docs/js/features/connectivity/bas-external-system.md

```diff
@@ -0,0 +1,165 @@
+---
+id: bas-external-system-js-sdk
+title: Connecting to external system from the business application studio (BAS) with the SAP Cloud SDK for JavaScript / TypeScript
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: From BAS to External Systems
+description: This article describes how the SDK helps to connect to external systems from the business application studio.
+keywords:
+- sap
+- cloud
+- sdk
+- proxy
+- connectivity
+- business application studio
+- cloud-foundry
+- JavaScript
+- TypeScript
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+## Introduction ##
+
+The business application studio (BAS) is a development environment offered as a service on cloud foundry (CF).
+You can simple subscribe to the BSA and from there start developing quickly without installing node, git, visual studio vode... .
+From the look and feel it is very similar to visual studio code, which is no sunrise since it is base on the [Eclipse Theia](https://theia-ide.org/) the open source version of visula studio code.
+
+However, SAP added a few useful features to the BAS. 
+You can connect to your CF account and access environment variables and destinations of the destination service subscribed to your account.
+This connection is also the focus of this document:
+How to reach external systems from your locally running BAS application without deploying to CF.
+This is a very useful feature for developers because the cycle to test something is much quicker if you can do it locally.
+We will also consider onPremise S/4 systems as relevant external system here, because for external systems reachable on the internet a connection is trivial.
```

Comment:
```suggestion
We will also consider SAP S/4HANA On-Premise systems as relevant external system here, because for external systems reachable on the internet a connection is trivial.
```